### PR TITLE
Added code for atm code

### PR DESCRIPTION
**Pull request by srisatya1197**
ma'am/Sir I added the code for issue-128 please review it . Let me know if there's is any changes I need to add

### CODE REVIEW

**Comment by srisatya1197** [comment]
ma'am Please review it when your free but if you check mine fast then I can add another code into it

**Comment by srisatya1197** [comment]
ma'am please check my pr because i made the new folder as per the your request and moreover i need to add some more codes into that folder

File: codechef/Beginner/ATM.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# while taking input we use input() function .
+
+a,b=input().split()  # Here I used a as 'x' and b as 'y' . 'a' stand for money and 'y' stands for how much balance does pooja have
+a=int(a) #Here the 'a' value is int if we observe the examples clearly we can understand
+b=float(b) # But the  'b' value is float why? because in the 'OUTPUT' it's mentioned that 'y' is of float type with two digits precision
+if a%5==0 and a+0.50<=b: # for finding remaining balance
+    rem_bal=b-a-0.50
+    print('%.2f'%rem_bal)
+else:
+    print('%.2f'%b) # while printing the output use two digit precision even if you are printing b in the else part.
```
Comment (rohanbabbar04) [inline codechef/Beginner/ATM.py:75]: Remove all the comments

File: codechef/Beginner/ATM.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+#NOTE : Some points to think of while writing the code is , as python follows indentation while taking input and while writing code ,
+# just be sure to write .spilt() function .The function .split() takes same line next input i.e., 1 2 in this way . If you do not
+# mention then while passing test cases code shows error . so as to overcome this situation use spilt() . Anyway for clarification
+# use with .split() and normally i.e., a=int(input()) b=float(input()) then you can understand inmportance of the function.
```
Comment (rohanbabbar04) [inline codechef/Beginner/ATM.py:81]: Remove the comments

File: codechef/Beginner/ATM.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+#
+# Output:
+# 120.00
+
+#Example - Insufficient Funds
+#Input:
+#300 120.00
+
+#Output:
+#120.00
```
Comment (rohanbabbar04) [inline codechef/Beginner/ATM.py:57]: Add sample output after the entire code 
Use docstrings instead of #

File: codechef/Beginner/ATM.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+# 4) write the code what you understood rather than directly entering to answer
+
+
+
+
+# first line : There's one girl , she want to withdraw some money ((money = "X" $US)) . Here , X must be a multiple of "5" . X can be
+#  5 10 15 ..............2000 . As, the limit of x is upto 2000 .
+# second line : The girl is having enough money for the withdrawal including bank charges . In the next let's see what exactly is charges
+# third line : For each successful withdrawal the bank charges 0.50 $US . Tru to understand as it is very important in the code
+# So , the bank charges will be deducted if and only if there is a successful withdrawal i.e., if x is a multiple of 5 .
```
Comment (rohanbabbar04) [inline codechef/Beginner/ATM.py:35]: Remove this

File: codechef/Beginner/ATM.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+#Output
+#Output the account balance after the attempted transaction, given as a number with two digits of precision.
+#If there is not enough money in the account to complete the transaction, output the current bank balance.
+
+
+#-------------------------------------------------------------------------------------------------------------------------------------
+
+
+
+
```
Comment (rohanbabbar04) [inline codechef/Beginner/ATM.py:20]: Remove extra lines

File: codechef/Beginner/ATM.py [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+#and Pooja's account balance has enough cash to perform the withdrawal transaction (including bank charges)   ---(read next)
+#For each successful withdrawal the bank charges 0.50 $US. Calculate Pooja's account balance after an attempted transaction.
+#Input
+#Positive integer 0 < X <= 2000 - the amount of cash which Pooja wishes to withdraw.
+
+#Nonnegative number 0<= Y <= 2000 with two digits of precision - Pooja's initial account balance.
+
+#Output
+#Output the account balance after the attempted transaction, given as a number with two digits of precision.
+#If there is not enough money in the account to complete the transaction, output the current bank balance.
```
Comment (rohanbabbar04) [inline codechef/Beginner/ATM.py:13]: Use docstrings for multi-line comments

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]
Shorten the code as much as possible
Also @srisatya1197 you said you added a new folder but it is not visible

**Comment by rcoder23** [comment]
@srisatya1197 Mention issue no.

File: codechef/Beginner/ATM.py (hunk 2 above)
Comment (srisatya1197) [reply codechef/Beginner/ATM.py:81]: Ok will do it Sir

File: codechef/Beginner/ATM.py (hunk 3 above)
Comment (srisatya1197) [reply codechef/Beginner/ATM.py:57]: Ok Sir

File: codechef/Beginner/ATM.py (hunk 4 above)
Comment (srisatya1197) [reply codechef/Beginner/ATM.py:35]: Ok Sir

File: codechef/Beginner/ATM.py (hunk 5 above)
Comment (srisatya1197) [reply codechef/Beginner/ATM.py:20]: Ok Sir

File: codechef/Beginner/ATM.py (hunk 6 above)
Comment (srisatya1197) [reply codechef/Beginner/ATM.py:13]: Ok Sir

**Comment by srisatya1197** [comment]
@rcoder23 ok Sir

**Comment by srisatya1197** [comment]
@rohanbabbar04 i made  a new folder codechef in it i made a new directory beginner and then i added my code into it

**Comment by srisatya1197** [comment]
@rohanbabbar04 Sir i made the changes should i write git add . or any other command

**Comment by srisatya1197** [comment]
@rohanbabbar04 Sir is the new folder visible ?

**Comment by srisatya1197** [comment]
@rcoder23 I'm sorry to ask this question but I want to .@rohanbabbar04  is he a participant or mentor .Just now I saw his nickname in discord there is P after his name ?

**Comment by rcoder23** [comment]
> @rcoder23 I'm sorry to ask this question but I want to .@rohanbabbar04 is he a participant or mentor .Just now I saw his nickname in discord there is P after his name ?

He is participate .

**Comment by srisatya1197** [comment]
Ok Sir. Anyway I made the changes

**Comment by rcoder23** [comment]
@srisatya1197  add Atm solution code in topic wise folder >>  codechef folder only

**Comment by srisatya1197** [comment]
Sir I created a new folder and in it I mentioned beginner into it Can you see it

**Comment by srisatya1197** [comment]
> @srisatya1197 add Atm solution code in topic wise folder >> codechef folder only

Yes Sir I did

**Comment by srisatya1197** [comment]
Sir I  made the changes but I'm unable to see them here

**Comment by rohanbabbar04** [comment]
@srisatya1197 
Did you do 
git add .
git commit -m "message"
git push origin branchname

**Comment by srisatya1197** [comment]
Yes I did

**Comment by rohanbabbar04** [comment]
Are you able to see it in your forked repo

**Comment by srisatya1197** [comment]
Lemme check

**Comment by srisatya1197** [comment]
I made the changes why it not showing?

**Comment by srisatya1197** [comment]
I will close this will make the new pr this is so puzzling me

**Pull request closed without merging**